src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -75,3 +75,8 @@
 #ifdef CONFIG_CHIP_ENABLE_RENDEZVOUS_BYPASS
 #define CHIP_BYPASS_RENDEZVOUS CONFIG_CHIP_ENABLE_RENDEZVOUS_BYPASS
 #endif // CONFIG_CHIP_ENABLE_RENDEZVOUS_BYPASS
+
+#ifdef CONFIG_CHIP_ENABLE_SERVICE_REGISTRATION
+#define CHIP_DEVICE_CONFIG_ENABLE_MDNS 1
```

Comment:
Hmm.. does `CHIP_DEVICE_CONFIG_ENABLE_MDNS` needs to be defined? Maybe it should be renamed to sth like `DNSSD` and `MDNS` where applicable - but I haven't looked deeply in the code.

Reply:
Unfortunately yes, we decided with Andrei for now to make from SRP something like service discovery implementation for MDNS. The goal is to reuse as much MDNS code and structures as possible and keep it consistent. I guess it is temporary solution to introduce SRP in CHIP, as soon as possible, but I talked with Damian that we should plan to refactor it in next sprints and create more generic module.

Reply:
Okay.